hack/build-manifests.sh

```diff
@@ -390,6 +383,11 @@ ${PROJECT_ROOT}/tools/csv-merger/csv-merger \
   --operator-image-name="${HCO_OPERATOR_IMAGE}" \
   --webhook-image-name="${HCO_WEBHOOK_IMAGE}" > "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}"
 
+rendered_csv="$(cat "${CSV_DIR}/${OPERATOR_NAME}.v${CSV_VERSION}.${CSV_EXT}")"
+rendered_keywords="$(echo "$rendered_csv" |grep 'keywords' -A 3)"
```

Comment:
I think `-A 3` argument on grep is not robust enough, as it will break if the number of keywords will be changed.